DigitalOffice/Tests/CompanyServiceTests/Repositories/CompanyRepositoryTests.cs

```diff
@@ -40,7 +61,26 @@ public void CleanDb()
                 dbContext.Database.EnsureDeleted();
             }
         }
+      
+        #region EditPosition
+        [Test]
+        public void SuccessfullyEditPosition()
+        {
+            newPosition = new DbPosition
+            {
+                Id = positionId,
+                Name = "abracadabra",
+                Description = "bluhbluh"
+            };
 
+            var result = repository.EditPosition(newPosition);
+            DbPosition newDbPosition = dbContext.Positions.FirstOrDefault(dbPosition => dbPosition.Id == positionId);
+
+            Assert.IsTrue(result);
+            SerializerAssert.AreEqual(newPosition, newDbPosition);
```

Comment:
It would be great to check the database: `Assert.That(dbContext.Positions, IsEquivalentTo(new List<DbPosition>{newDbPosition}`

Reply:
But I'm checking it in newDbPosition variable and using SerializerAssert

Reply:
You are checking the position change, but not the db state.

Reply:
Fixed